### DISCUSSION TITLE
Detailed description for developers on Windows 10

### DISCUSSION THREAD

**Discussion by h8nor** [serkonda7/vscode-vba#109] [Q&A] [answered]
1. First need to install the NVM to control the Node.js&reg; versions.
   ```cmd
   mkdir %APPDATA%\nvm & cd %APPDATA%\nvm
   curl -OL https://github.com/coreybutler/nvm-windows/releases/latest/download/nvm-setup.zip
   tar -xf nvm-setup.zip && start /wait nvm-setup.exe && install
   ```
   _The file `settings.txt` is copied into the root of disk `C:\`_
2. Install your chosen version of Node.js&reg;
   ```cmd
   nvm install 18.19.0 && nvm use 18.19.0
   ```
3. Open a path `./vscode-vba` in console. Install the components. Clone repository of the **Bait Programming Language**.
   ```bash
   git merge --allow-unrelated-histories upstream
   
   pnpm -v || npm i -g pnpm
   pnpm install
   
   git clone https://github.com/bait-lang/bait dist/bait
   ```
4. Open a path `./vscode-vba/dist/bait` in console and follow [instructions](https://github.com/bait-lang/bait#installation).  
   _First command must creating the file_ `bait.js`.
5. Necessarily restart terminal or reboot Windows.
   Make changes and [Install a dev version](https://github.com/serkonda7/vscode-vba/blob/main/CONTRIBUTING.md#install-a-dev-version).

> **Bonus.** Add to `keybindings.js` **vscode** to call the Inspector through a hotkey.
> ```json
> [
>     {
>         "key": "f8",
>         "command": "editor.action.inspectTMScopes"
>     }
> ]
> ```

**Comment by serkonda7** [comment, marked as answer]
With #110, the contribution guideline should be improved and reflect most of your points.
